sirepo/server.py

```diff
@@ -432,6 +432,20 @@ async def api_saveSimulationData(self):
             ),
         )
 
+    @sirepo.quest.Spec("allow_visitor")
+    async def api_securityTxt(self):
+        d = sirepo.srtime.utc_now()
+        d = d.replace(year=d.year + 1, hour=0, minute=0, second=0, microsecond=0)
```

Comment:
This doesn't work right. On 12/31/2024, the server will return 1/1/2025. The server will then return expiry times that are in the past, especially if there is no prod release around 12/25, which is likely. Is there a reason to force to a specfic date/time? If so, set it to +2 years.